path_supplier/graphhopper_path_supplier.go

```diff
@@ -0,0 +1,92 @@
+package path_supplier
+
+import (
+	"encoding/json"
+	"fmt"
+	m "github.com/harfmuf/geolocation-generator/model"
+	"io/ioutil"
+	"net/http"
+)
+
+// precision to around .0001 deg, which corresponds to max 10m
+const URL_FORMAT = "%s%s?point=%.4f,%.4f&point=%.4f,%.4f&vehicle=%s&key=%s&points_encoded=false"
+const GRAPHHOPPER_API_URL_BASE = "https://graphhopper.com/api/1"
+const ROUTING_API_URL = "/route"
+
+type GraphHopperPathSuplier struct {
+	baseUrl string
+	apiKey  string
+}
+
+type GraphHopperResponse struct {
+	Paths []Path
+}
+
+type Path struct {
+	Points Points
+}
+
+type Points struct {
+	Coordinates [][]float64
+}
+
+func New(baseUrl string, apiKey string) *GraphHopperPathSuplier {
+	return &GraphHopperPathSuplier{apiKey: apiKey, baseUrl: baseUrl}
+}
+
+func (g GraphHopperPathSuplier) FindPath(from m.Location, to m.Location, vehicle string) [][]m.Location {
```

Comment:
Why parameters and result are not pointers?

Reply:
Done